docs/required-fields.md

```diff
@@ -1,30 +1,46 @@
+
+Before you begin, we strongly advise that you follow Operator-Lifecycle-Manager's docs on [building a CSV for the Operator Framework](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/building-your-csv.md). These outline the functional purpose of the CSV and which fields are required for installing your Operator CSV through OLM.
+
+## Required fields for OperatorHub.io
+
 An Operator's CSV must contain the following annotations for it to be displayed properly within OperatorHub.io. A full example is shown at the bottom of this document.
 
 ```yaml
 metadata:
   annotations:
-    capabilities: The operator's maturity level. This field can be omitted until a process to get this values is defined at a later date.
-    categories: A list of comma separated list of categories from the values below.
-    certified: The operator's certification. This field should be set to false until a process to get this value is defined at a later date.
+    capabilities: One of the following: Basic Install, Seamless Upgrades, Full Lifecycle, Deep Insights, Auto Pilot. For more information see https://www.operatorhub.io/images/capability-level-diagram.svg
+    categories: A list of comma separated list of categories from the values below. If not set, this will be set to "Other" in the UI.
     description: |-
       A short description of the operator that will be displayed on the marketplace tile.
       If this annotation is not present, the `spec.description` value will be shown instead.
       In either case, only the first 135 characters will appear.
     containerImage: The repository that hosts the operator image. The format should match ${REGISTRYHOST}/${USERNAME}/${NAME}:${TAG}
     createdAt: The date that the operator was created. The format should match yyyy-mm-ddThh:mm:ssZ
-    support: The maintainer of the operator. This value should be a name, not an email.
+    repository: Optional. The URL operator's source code repository.
+    alm-examples: A string of a JSON list of example CRs for the operator's CRDs. These should be able to work without modification, unless otherwise stated in the operator description.
 spec:
-  displayName: A short name for the operator.
+  displayName: A short, readable name for the operator.
   description: A detailed description of the operator, preferably in markdown format.
   icon: A base 64 representation of an image associated with your operator
-  version: The operator version.
+  version: The operator version in semver format.
+  maintainers:
+  - name: Name of a maintainer of the operator
+    email: Email to reach maintainer
   provider:
     name: The provider of the operator. This value should be a name, not an email.
+  links:
+  - name: Title of the link (ex: Blog, Source Code etc.)
+    url: url/link
+  keywords:
+  - 'A list of words that relate to your operator'
+  - 'These are used when searching for operators in the UI'
```

Comment:
Adding this in https://github.com/operator-framework/operatorhub.io/pull/83

Reply:
Thanks @jeff-phillips-18 ! Looks like it's merged, so I'll go ahead with this PR.